src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage() {
+        super();
+    }
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "userInput")
+    private MobileElement userInput;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "passwordInput")
+    private MobileElement passwordInput;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "logInButton")
+    private MobileElement btnlogInButton;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.widget.TextView[@text='Forgot the password?']")
+    private MobileElement lnkForgotPassword;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.widget.EditText[@content-desc=\"passwordInput\"]/parent::android.view.ViewGroup/following-sibling::*[@class='android.widget.TextView']")
```

Comment:
What does this selects? i could't find it

Reply:
<img width="712" alt="ErrorCredenciales" src="https://user-images.githubusercontent.com/75274560/108864727-79a59f00-75c0-11eb-8bc8-1748a558d7fa.png">